### PR TITLE
Deploy project

### PR DESCRIPTION
**Pull request by ivangonzalez224**
It is the website for the Latin American Veganism event and I added the followings new features:

- Add the toolbar, headline, program, and partner sections html skeleton in home page.
- Add the speakers section dynamically in home page.
- Add the toolbar, headline, organization, and previous events sections html skeleton in about page.
- Add styles to the all the sections in home and about pages for mobile and desktop versions.
- Include images needed in the home and about pages.
- Add event listeners for the mobile version menu in home and about pages.

### CODE REVIEW

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - [ ] The provided design template has two navbar for the desktop view, which include the `top dark bar` & `bottom white bar` which are shown in the images below however your design does not have any of these navbar for desktop view. please i kindly suggest you add them to your project becasue it is important that your webpage matches closely the provided design :+1: :sparkles: 
- [ ] Make sure to add your logo as well as aligning the contents correctly as shown in the provided design template :+1: 
- design template
nav
- your design 
navv

Comment (Efiamotu-1) [inline README.md:1]: - [ ] You currently do not have a partner section and a footer for your desktop view. I have noticed from your project design that you ahev used the same design for mobible to represent the design for desktop however this is not the case as they both have different structure. Please see the [provided design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and make the necessary changes to the desktop view :+1:  
- design template
vv
- your design
bv

Comment (Efiamotu-1) [inline README.md:1]: - [ ] The home section of your webpage is not properly asligned on the desktop view as shown in the design template below, please make sure to check the spaqcing and alignment as shown and make the necessary changes before resubmitting :+1: 
- design template
dp
- your design
dp

Comment (Efiamotu-1) [inline README.md:1]: - [ ] As shown in the images below you are not using the provided design look for desktop screen sizes for displaying the main program please make the necessary corrections. :+1: 
- design template
mp
- your design
mps

Comment (Efiamotu-1) [inline README.md:1]: - [ ] Your featured speaker section for desktop screen as shwon  below is not consistent with the provided design template please closely match your design to show your proficiency with working with template designs as a professionall front-end developer :+1: 
- design template
sps
- your design
sp

Comment (Efiamotu-1) [inline README.md:1]: - [ ] On mobile view, there is an extra layer of space as shown by the red circle. i kindly suggest you work on removing it soy our design looks good for mobile screens
- your design
ext

Comment (Efiamotu-1) [inline README.md:1]: - [ ] Your about page is also missing the two navbars as can be seen in the design template below
- design template
vb
- your design
 abb

Comment (Efiamotu-1) [inline README.md:1]: - [ ] The below image is the provided design for the bottom section and footer for the about page. currently your project doesn't follow this strucure for the about page on desktop view so i kindly suggest you work on fixing your design to match closely the provided design template :+1: 
- design template
d-ab

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @ivangonzalez224  

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ivangonzalez224** [comment]
For this second review request, I am attaching the link to the video I recorded at the request of the previous reviewer. The previous reviewer had a particular issue. First, he was seeing the mobile version in his browser when he had a screen in the desktop version, secondly, in his browser the two navbar for the desktop view were no working well, which include the top dark bar & bottom white bar, and for those reasons he asked me to record a video showing the web pages to verify that my project works well.
- Video link showing the web pages working well: https://www.loom.com/share/8005940bcbea46c4b7efea96fc15363f?utm_medium=gif

**Review by sumairq — APPROVED** [review]
### Hi @ivangonzalez224 , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**